tools/terraform/variables.tf

```diff
@@ -60,7 +60,7 @@ variable "database_name" {
 
 variable "fleet_image" {
   description = "the name of the container image to run"
-  default     = "fleetdm/fleet"
+  default     = "fleetdm/fleet:v4.8.0"
```

Comment:
Is this something that we should be updating with each Fleet release?

Reply:
I think so. Today tracking latest has had some folks get unexpected upgrades, and they weren't aware that migrations needed to be ran. This was a mistake on my part not locking it down to a specific version.